spec_42.txt

```diff
@@ -0,0 +1,179 @@
+
++++ Patch Requirements
+
+* Maintainers and Contributors MUST have a Platform account and SHOULD use their real names or a well-known alias.
+
+* A patch SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.
+
+* A patch MUST adhere to the code style guidelines of the project if these are defined.
+
+* A patch MUST adhere to the "Evolution of Public Contracts" guidelines defined below.
+
+* A patch SHALL NOT include non-trivial code from other projects unless the Contributor is the original author of that code.
+
+* A patch MUST compile cleanly and pass project self-tests on at least the principle target platform.
+
+* A patch commit message SHOULD consist of a single short (less than 50 character) line summarizing the change, optionally followed by a blank line and then a more thorough description.
+
+* A "Correct Patch" is one that satisfies the above requirements.
+
++++ Development Process
+
+* Change on the project SHALL be governed by the pattern of accurately identifying problems and applying minimal, accurate solutions to these problems.
+
+* To request changes, a user SHOULD log an issue on the project Platform issue tracker.
+
+* The user or Contributor SHOULD write the issue by describing the problem they face or observe.
+
+* The user or Contributor SHOULD seek consensus on the accuracy of their observation, and the value of solving the problem.
+
+* Users SHALL NOT log feature requests, ideas, suggestions, or any solutions to problems that are not explicitly documented and provable.
+
+* Thus, the release history of the project SHALL be a list of meaningful issues logged and solved.
+
+* To work on an issue, a Contributor SHALL fork the project repository and then work on their forked repository.
+
+* To submit a patch, a Contributor SHALL create a Platform pull request back to the project.
+
+* A Contributor SHALL NOT commit changes directly to the project.
+
+* If the Platform implements pull requests as issues, a Contributor MAY directly send a pull request without logging a separate issue.
+
+* To discuss a patch, people MAY comment on the Platform pull request, on the commit, or elsewhere.
+
+* To accept or reject a patch, a Maintainer SHALL use the Platform interface.
+
+* Maintainers SHOULD NOT merge their own patches except in exceptional cases, such as non-responsiveness from other Maintainers for an extended period (more than 1-2 days).
+
+* Maintainers SHALL NOT make value judgments on correct patches.
+
+* Maintainers SHALL merge correct patches from other Contributors rapidly.
+
+* Maintainers MAY merge incorrect patches from other Contributors with the goals of (a) ending fruitless discussions, (b) capturing toxic patches in the historical record, (c) engaging with the Contributor on improving their patch quality.
+
+* The Contributor MAY tag a user's issue as "Ready" after making a pull request for the issue.
```

Comment:
For the Contributor  this would imply changing the description of the pull request before merging which feels weird IMO. Tagging an issue with a "Ready" label does seem more appropriate if that's what this is referring to. Though I never did this in practices but I seems worth trying!